subsys/logging/Kconfig.formatting

```diff
@@ -58,6 +58,14 @@ config LOG_BACKEND_SHOW_COLOR
 	help
 	  When enabled selected backend prints errors in red and warning in yellow.
 
+if LOG_BACKEND_SHOW_COLOR
+
+config LOG_INFO_COLOR_GREEN
+	bool "Use green color for info level logs"
+	default n
```

Comment:
It's superfluous because boolean symbols don't need a default and will default to n.